src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
```

Comment:
I think comment in L11 can present `getStatusIcon()` function. So how about moving the comment to the above of function. For example, 

```Java
// bla bla
public String getStatusIcon() {

}
```

Reply:
Thank you for all your feedback! It was very useful, and I will make the changes accordingly! 👍